<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="INBNRMARITA" failures="1" tests="5" name="Close PO" time="30.998" errors="0" timestamp="2022-04-26T12:03:29 IST">
  <testcase classname="script.NewClosePoLogistics" name="poClose" time="10.072"/>
  <testcase classname="script.NewClosePoLogistics" name="resetButton" time="2.038"/>
  <testcase classname="script.NewClosePoLogistics" name="validateCropList" time="2.088"/>
  <testcase classname="script.NewClosePoLogistics" name="companyName" time="0.037"/>
  <testcase classname="script.NewClosePoLogistics" name="closePO" time="12.144">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at script.NewClosePoLogistics.closePO(NewClosePoLogistics.java:95)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- closePO -->
</testsuite> <!-- Close PO -->
